packages/gatsby-theme-newrelic/src/components/CookieConsentDialog.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Cookies from 'js-cookie';
+import { css } from '@emotion/core';
+import ExternalLink from './ExternalLink';
+import Button from './Button';
+
+const CookieConsentDialog = ({
+  cookieName,
```

Comment:
Since we are moving this to the theme, I think we should really be opinionated for this cookie dialog since it is very specific to New Relic. That means not leaving the cookie name up to consumers of this component, but rather we will set it for them.

I really like that we are pulling the cookie consent into the theme (its loongggg overdue). In doing so, I think we should strive for as minimal configuration as possible. This is one of those values that is not really going to need to be tweaked site-to-site.

Reply:
I agree! I'll make those changes